king_phisher/client/assistants/campaign.py

```diff
@@ -127,6 +172,37 @@ def __init__(self, application, campaign_id=None):
 			self.gobjects['label_intro_body'].set_text('This assistant will walk you through creating and configuring a new King Phisher campaign.')
 			self.gobjects['label_intro_title'].set_text('New Campaign')
 
+		self._url_selected = None
```

Comment:
This shouldn't need to be stored. You should get the selected row from the treeview when you need the value.